Mathlib/Tactic/ToAdditive.lean

```diff
@@ -1031,6 +1031,8 @@ def fixAbbreviation : List String → List String
                                       => "function" :: "_" :: "commute" :: fixAbbreviation s
   | "zero" :: "Le" :: "Part" :: s         => "posPart" :: fixAbbreviation s
   | "le" :: "Zero" :: "Part" :: s         => "negPart" :: fixAbbreviation s
+  | "three" :: "GPFree" :: s         => "three" :: "APFree" :: fixAbbreviation s
+  | "Three" :: "GPFree" :: s         => "Three" :: "APFree" :: fixAbbreviation s
```

Comment:
Does `GPFree` or `gpfree` occur as a atomic part of any name that should not be translated to `APFree`/`apfree`? If there are no such cases, you can you just add `| "gpfree"=> ["apfree"]` to `nameDict`. That is the preferred way to translate "easy" cases.